code/datums/EPv2.dm

```diff
@@ -105,23 +105,23 @@ var/global/list/all_exonet_connections = list()
 // Proc: send_message()
 // Parameters: 2 (target_address - the desired address to send the message to, message - the message to send)
 // Description: Sends the message to target_address, by calling receive_message() on the desired datum.
-/datum/exonet_protocol/proc/send_message(var/target_address, var/message)
+/datum/exonet_protocol/proc/send_message(var/target_address, var/message, var/text)
```

Comment:
If you're going to add another argument, can you update the comment above it telling everyone what it does?